sympy/series/formal.py

```diff
@@ -982,12 +995,16 @@ def _get_pow_x(self, term):
     def polynomial(self, n=6):
         """Truncated series as polynomial.
 
-        Returns series sexpansion of ``f`` upto order ``O(x**n)``
+        Returns series expansion of ``f`` upto order ``O(x**n)``
         as a polynomial(without ``O`` term).
         """
         terms = []
         for i, t in enumerate(self):
+            t = powsimp(t)
```

Comment:
Can this be moved inside ``_get_pow_x`` instead?

Reply:
I don't think we need this anymore.